internal/local/environment_test.go

```diff
@@ -0,0 +1,29 @@
+package local
+
+import (
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/cli/user"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/10gen/realm-cli/internal/utils/test/mock"
+)
+
+func TestLoadProfiles(t *testing.T) {
+	t.Run("should return a list of profile names, file paths and api keys", func(t *testing.T) {
```

Comment:
[nit] ditch the commas in the test description

Reply:
is this just a styling preference or does it actually mess with the tests?

Reply:
styling preference... the `,` right before the `_` (which replaces the space) in `go test...` output makes it a bit clunkier to read